data/_ui/tooltips.txt

```diff
@@ -1092,23 +1092,242 @@ tip "This outfit is considered an atrocity."
 
 
 # Controls
-tip `Reverse`
-	`Turn opposite to the flight direction (or use reverse engines if you have them installed). Combine with <shift> key to stop your ship.`
+tip "Forward thrust"
+	`Accelerate your flagship forward.`
+
+tip "Turn left"
+	`Turn your flagship left.`
+
+tip "Turn right"
+	`Turn your flagship right.`
+
+tip "Reverse"
+	`Turn your flagship opposite its current direction of flight. If reverse thrusters are installed, accelerate backwards. With <shift>, stop your ship.`
```

Comment:
I don't think information about the behavior of the halt command is relevant to the tooltip. All the player needs to know is that this will halt their ship.